package.json

```diff
@@ -49,8 +49,8 @@
     "eslint-plugin-unicorn": "^16.1.1",
     "fs-extra": "^8.1.0",
     "husky": "^4.2.5",
-    "jest": "^25.4.0",
     "jest-ci": "^0.1.1",
+    "jest-cli": "^25.5.4",
```

Comment:
Why did you add this change?

Reply:
Tests were not passing me, not sure if by using `npm` instead of `yarn`. They gave me an error of not being able to find a file inside `jest-cli` package, probably because you have `jest-cli` installed globally. I've check that in the dependencies is being used `jest`, while it's a placeholder package and the recommended one is in fact `jest-cli`, that's why I've changed it.

Reply:
> probably because you have jest-cli installed globally

Nope. I do not install npm package globally (I think it is a very bad practice)

Reply:
Commands were recommended to be installed globally, but except for particular use cases, yes, now it's recommended to install them locally.